CRM/Admin/Form/SettingTrait.php

```diff
@@ -171,9 +171,13 @@ protected function addFieldsDefinedInSettingsMetadata() {
         elseif ($add == 'addMonthDay') {
           $this->add('date', $setting, ts($props['title']), CRM_Core_SelectValues::date(NULL, 'M d'));
         }
+        elseif ($add === 'addYesNo' && ($props['type'] === 'Boolean')) {
+          $this->addRadio($setting, ts($props['title']), array(0 => 'No', 1 => 'Yes'), NULL, '&nbsp;&nbsp;');
```

Comment:
Most YesNo fields are displayed with Yes as the first element (see Settings -> Search Preferences and custom fields of type YesNo).
Can we flip this array so Yes is shown first on the UI?

Reply:
done